webpack.config.js

```diff
@@ -72,7 +72,7 @@ module.exports = {
         use: {
           loader: 'svg-url-loader',
           options : {
-            publicPath: '/',
+            publicPath: process.env.PUBLIC_PATH || '/',
```

Comment:
I think if you do this it'll just try to serve the SVGs from the static domain again, and that'll break because of security checks.